ts/state/selectors/messages.ts

```diff
@@ -34,12 +35,13 @@ export const useAuthorProfileName = (messageId: string): string | null => {
   if (!msg || !senderProps) {
     return null;
   }
+  const { sender } = msg.propsForMessage;
 
-  const senderIsUs = msg.propsForMessage.sender === UserUtils.getOurPubKeyStrFromCache();
+  const senderIsUs = sender === UserUtils.getOurPubKeyStrFromCache();
```

Comment:
Should we not use `useIsMe`? here since we are in a hook?

Reply:
fixed